Swiftfin/Views/SettingsView/UserDashboardView/DevicesView/DevicesView.swift

```diff
@@ -0,0 +1,201 @@
+//
+// Swiftfin is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2024 Jellyfin & Jellyfin Contributors
+//
+
+import Defaults
+import JellyfinAPI
+import SwiftUI
+
+struct DevicesView: View {
```

Comment:
I was originally going to do a DeviceDetailView but I was finding that the only fields that weren't in use from the DeviceView we're very worthwhile:

<img width="463" alt="Screenshot 2024-10-16 at 08 54 05" src="https://github.com/user-attachments/assets/6dfa33fd-f841-4125-a73f-0156fb62555d">

I'd still be happy building out a DeviceDetailView but I think it'd be the same information + once we the SDK updates to include customName (its on main now) I can add that as well. 

Your call!

Reply:
> We could also use the last used user id and show basic user information.

I'm adding some changes to this commit that are for a later PR:

One of the other items I have built out in another branch is the Users section. That would show all Jellyfin Users. 

The UserDetails has Forms for changing Password & Permissions, and a listView for Devices. That calls this same view but filters on UserID.

So I was thinking instead of going Devices > Users we could go from Users > Devices then have this same view reused there to allow changes to specific user devices.

Reply:
Even if it's just a few more fields, I still prefer it over the alert viewing for changing the name.

Per the user comment, I'm not saying we should push to `Users` from this view. We could just get the single-user data and just show their profile picture and name like we do in sessions. Then, we should push to the user detail view from there.

Reply:
Sounds good! Does this look like everything we want?

![Simulator Screenshot - iPhone 16 Pro - 2024-10-16 at 12 27 22](https://github.com/user-attachments/assets/2f3f906e-e61e-4d08-ab64-6e0457587951)

Reply:
That looks great!